atomic_reactor/constants.py

```diff
@@ -189,3 +192,21 @@
 
 # location for build info directory in the image
 IMAGE_BUILD_INFO_DIR = '/root/buildinfo/'
+
+
+UserConfigFile = namedtuple('UserConfigFile', ['filename', 'schema'])
+
+USER_CONFIG_FILES = {
```

Comment:
why not have just simple dict 
{REPO_FETCH_ARTIFACTS_URL: 'schemas/fetch-artifacts-url.json', .... } 
?

having the same value in key and then again in filename seems a waste